table/tables/index.go

```diff
@@ -226,11 +227,26 @@ func (c *index) Create(sctx sessionctx.Context, txn kv.Transaction, indexedValue
 	}
 	if err != nil || len(value) == 0 {
 		lazyCheck := sctx.GetSessionVars().LazyCheckKeyNotExists() && err != nil
+		var needPresumeKey int
```

Comment:
It's strange that needPresumeKey is an integer.

Reply:
TestCreateUniqueExpressionIndex